crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1048,29 +1010,35 @@ pub fn prepare_mesh_bind_group(
     skins_uniform: Res<SkinUniform>,
     weights_uniform: Res<MorphUniform>,
 ) {
-    groups.reset();
-    let layouts = &mesh_pipeline.mesh_layouts;
     let Some(model) = mesh_uniforms.binding() else {
         return;
     };
-    groups.model_only = Some(layouts.model_only(&render_device, &model));
+    let layouts = &mesh_pipeline.mesh_layouts;
+    let mut bind_groups = groups.bind_group_builder(&render_device, model, layouts);
 
-    let skin = skins_uniform.buffer.buffer();
-    if let Some(skin) = skin {
-        groups.skinned = Some(layouts.skinned(&render_device, &model, skin));
-    }
+    // no morph + maybe(skin) * motion_vectors + no motion_vectors
```

Comment:
It's the "shader features" for the variants being added to the bind group (to re-use the terms I use in the `mesh_bindings.rs` comments).

Here it adds the bind groups for (mv = motion vectors):

- !morph × !skin × !mv
- !morph × skin × !mv
- !morph × skin × mv

So "morph: no, skin: yes/no, motion_vectors: yes/no. The second comment is similar, but with always morph:yes.

 But it might be more confusing than anything really. Plus the `add_variant` method call says as much really.

Reply:
Why `+` and `*`? :)

Reply:
But do you understand what I'm getting at? Is it fine to remove the comment (I removed it)